Formula/fastd.rb

```diff
@@ -26,21 +26,35 @@ class Fastd < Formula
   end
 
   depends_on "cmake" => :build
-  depends_on "libuecc"
-  depends_on "libsodium"
   depends_on "bison" => :build # fastd requires bison >= 2.5
   depends_on "pkg-config" => :build
   depends_on "json-c"
+  depends_on "libuecc"
   depends_on "openssl" => :optional
   depends_on :tuntap => :optional
 
+  # https://projects.universe-factory.net/issues/251
+  resource "libsodium" do
+    url "https://github.com/jedisct1/libsodium/releases/download/1.0.14/libsodium-1.0.14.tar.gz"
+    sha256 "3cfc84d097fdc891b40d291f2ac2c3f99f71a87e36b20cc755c6fa0e97a77ee7"
+  end
+
   def install
     args = std_cmake_args
     args << "-DENABLE_LTO=ON"
     args << "-DENABLE_OPENSSL=ON" if build.with? "openssl"
-    args << buildpath
+
+    resource("libsodium").stage do
+      system "./configure", "--disable-debug", "--disable-dependency-tracking",
+                            "--prefix=#{libexec}/libsodium"
+      system "make", "check"
+      system "make", "install"
+    end
+    args << "-DLIBSODIUM_INCLUDE_DIR=#{libexec}/libsodium/include"
+    args << "-DLIBSODIUM_LIBRARIES=#{libexec}/libsodium/lib/libsodium.dylib"
+
     mkdir "fastd-build" do
-      system "cmake", *args
+      system "cmake", buildpath, *args
```

Comment:
".."

Reply:
I just dropped a cut/paste job here, but sure, can tweak it to `..` instead.